documentation/modules/metrics/proc-metrics-cruise-control.adoc

```diff
@@ -0,0 +1,62 @@
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly-metrics.adoc
+
+[id='cruise-control-monitoring-{context}']
+= Presenting Cruise Control metrics in Grafana
+
+[role="_abstract"]
+If you are using Prometheus and Grafana to monitor built-in Kafka metrics,
+you can configure Prometheus to also scrape the Cruise Control endpoint.
+You can then present the Cruise Control metrics in Grafana.
+
+Strimzi provides xref:ref-metrics-dashboards-{context}[example Grafana dashboard configuration files for Cruise Control].
+The following Cruise Control dashboard is provided as a JSON file:
+
+* `strimzi-cruise-control.json`
+
+This procedure uses the example dashboard to present metrics data in Grafana.
+
+.Prerequisites
+* xref:assembly-metrics-prometheus-{context}[Prometheus and Prometheus Alertmanager are deployed]
+* xref:proc-metrics-grafana-dashboard-{context}[Grafana is deployed and connected with Prometheus]
+
+.Procedure
+
+. Enable Cruise Control metrics using the `cruiseControl.metricsConfig` property in the `Kafka` resource to provide a reference to a ConfigMap that contains JMX exporter configuration for the metrics to expose.
++
+For example:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  kafka:
+    # ...
+  zookeeper:
+    # ...
+  cruiseControl:
+    metricsConfig:
+       type: jmxPrometheusExporter
+       valueFrom:
+         configMapKeyRef:
+           name: my-config-map
+           key: my-key
+----
```

Comment:
It seems a bit weird to have this here without any further reference. If nothing else, this should link to the example so that readers know what should be in the config map.

I also wonder again about the use-case of the procedure. It talks about Grafana from the title. But do we have other procedures talking about the Prometheus endpoint part? If so, why split this into separate procedures. If this is the only procedure ... the title seems misleading - it should talk more about metrics and Prometheus

Reply:
File removed